app/helpers/application_helper.rb

```diff
@@ -12,6 +12,10 @@ def create_event_or_login(name, classes)
     end
   end
 
+  def become_a_manager(name, classes)
+    content_tag(:a, name, href:new_manager_profile_path, class: classes*' ' )
```

Comment:
Space missing after colon.<br>Surrounding space missing for operator `*`.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.